bug: tfsec crashes since last saturday

Can't figure out on which resource exactly, but one of azure keyvaults.

```
panic: not a string

goroutine 1 [running]:
github.com/zclconf/go-cty/cty.Value.AsString({{{0x0, 0x0}}, {0x0, 0x0}})
	/home/runner/work/tfsec/tfsec/vendor/github.com/zclconf/go-cty/cty/value_ops.go:1256 +0x138
github.com/aquasecurity/tfsec/internal/pkg/adapter/azure/keyvault.adaptSecret(0xc000823220)
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/azure/keyvault/adapt.go:113 +0x165
github.com/aquasecurity/tfsec/internal/pkg/adapter/azure/keyvault.(*adapter).adaptVaults(0xc0025f6130, {0xc000980800, 0xbbdc07, 0x80})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/azure/keyvault/adapt.go:44 +0x347
github.com/aquasecurity/tfsec/internal/pkg/adapter/azure/keyvault.Adapt({0xc000980800, 0x45, 0x80})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/azure/keyvault/adapt.go:18 +0x11c
github.com/aquasecurity/tfsec/internal/pkg/adapter/azure.Adapt({_, _, _})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/azure/adapt.go:30 +0x356
github.com/aquasecurity/tfsec/internal/pkg/adapter.Adapt({0xc000980800, 0x45, 0x80})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/adapt.go:20 +0x7e
github.com/aquasecurity/tfsec/internal/pkg/scanner.(*Scanner).Scan(0xc001ad7080, {0xc000980800, 0x0, 0x80})
	/home/runner/work/tfsec/tfsec/internal/pkg/scanner/scanner.go:51 +0x9f
github.com/aquasecurity/tfsec/internal/app/tfsec/cmd.glob..func1(0x12801a0, {0xc0003e8720, 0x1, 0x6})
	/home/runner/work/tfsec/tfsec/internal/app/tfsec/cmd/root.go:114 +0x7ec
github.com/spf13/cobra.(*Command).execute(0x12801a0, {0xc00001e080, 0x6, 0x6})
	/home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x12801a0)
	/home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:902
main.main()
	/home/runner/work/tfsec/tfsec/cmd/tfsec/main.go:11 +0x25
##[error]Bash exited with code '2'.

```

--- a/internal/pkg/adapter/azure/keyvault/adapt.go
+++ b/internal/pkg/adapter/azure/keyvault/adapt.go
@@ -109,7 +109,7 @@ func adaptSecret(resource *block.Block) keyvault.Secret {
 	expiryDateAttr := resource.GetAttribute("expiration_date")
 	expiryDateVal := types.TimeDefault(time.Time{}, resource.GetMetadata())
 
-	if expiryDateAttr.IsNotNil() {
+	if expiryDateAttr.IsString() {
 		expiryDateString := expiryDateAttr.Value().AsString()
 		layout := "2006-01-02T15:04:00Z"
 		if expiryDate, err := time.Parse(layout, expiryDateString); err == nil {

